lib/screens/HomePage.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+
+class HomePage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text(""),
+        backgroundColor: Colors.blueGrey,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            Center(
+              child: Text(
+                "Posts: 1000  Followers: 50000  Following: 1000",
+                style: TextStyle(fontSize: 16),
+              ),
+            ),
+            // SizedBox(height: 10), // Remove or adjust this SizedBox
+            Container(
+              width: 150,
+              height: 150,
+              decoration: BoxDecoration(
+                borderRadius: BorderRadius.circular(20),
+                image: const DecorationImage(
+                  image: NetworkImage(
+                    "https://cdn-icons-png.freepik.com/512/3270/3270999.png",
+                  ),
+                ),
+              ),
+            ),
+            Center(
+              child: Text(
+                "@Fatimah M ",
+                style: TextStyle(fontSize: 16),
+              ),
+            ),
+            SizedBox(height: 10),
+            // Three larger images side by side
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
```

Comment:
```suggestion
            SizedBox(
                height: 200,
                child: ListView(
                  scrollDirection: Axis.horizontal,
```
replace your Row with sized box with a child listview the scrolls horizontally so it doesn't overflow on the side.